src/modal.js

```diff
@@ -48,7 +48,6 @@ class Modal extends Component {
     if (this.props.closeOnEsc) {
       document.removeEventListener('keydown', this.handleKeydown);
     }
-    document.body.style.overflow = this.state.previousBodyStyleOverflow;
```

Comment:
Not sure about removing this line if a modal is just destroyed before the open props is changed the overflow will be blocked. (User change a route when modal is open etc ..)